### PR TITLE
Update Ansible Variables

### PR DESCRIPTION
**Pull request by dmc5179**
Update Ansible variables to separate OCP provisioner variables in aws.yml from variables needed for the workshop to be deployed.

### CODE REVIEW

**Comment by dmc5179** [comment]
@ecwpz91 @gbengataylor

Can you take a quick look at these changes to make sure we're happy with them and ready to merge in? Thanks.

File: .gitignore [hunk 1]
```diff
@@ -2,18 +2,7 @@
...
-scripts/redeploy-sonarqube.sh
-scripts/provision-batch-setup-me.sh
-scripts/provision-batch-delete-me.sh
-scripts/provision-batch-setup-me-sonarqube.sh
-scripts/quay/provision-quay-me.sh
-scripts/quay/provision-quay-batch-me.sh
-scripts/quay/delete-quay-me.sh
-.gitignore
-
-vars/*
```
Comment (gbengataylor) [inline .gitignore:17]: I think we should put
```script
vars/*
!vars/aws.example.yml
```
back
it's common practice tend to create a vars file for each cluster an instructor is working on. by removing vars/*, each is then registered as a change
vars/aws.example.yml is what we use to document new changes

Also, what is aws.yml going to contain?

File: vars/devsecops.example.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
```
Comment (gbengataylor) [inline vars/devsecops.example.yml:1]: Is this file just a copy of aws.example.yml?

File: .gitignore (hunk 1 above)
Comment (dmc5179) [reply .gitignore:17]: @gbengataylor The workshop repository did several things:

1) The playbooks used aws.example.yml when what should be done, as is the case in Jarred's provisioner, the user copies aws.example.yml to aws.yml and makes their changes. I updated the main yml file to use aws.yml and not aws.example.yml

2) The aws.yml file blended what appears to be an older version of the aws.yml file from Jarred's provisioner with the variables specific to the workshop. I separated those variables out into the devsecops.example.yml. The user will then copy that file to devsecops.yml which they can change and is used by the primary playbook.

3) I updated the run script to source the aws.env file to set the AWS credentials.

The net effect of these changes is that the example files follow the same pattern as our other repos and by separating the workshop variables into their own file user can now simply make a symlink to the aws.yml and aws.env files they used to provision the cluster. Prior to these changes the user would have to copy changes from their aws.yml from cluster install into aws.example.yml

I can certainly add !vars/aws.example.yml back. If the repo is being used to deploy the workshop to multiple clusters my changes will allow them to simply link to the aws.yml file from each deployment.

**Pull request merged**